README.md

```diff
@@ -22,3 +22,12 @@ cp _redirects _site/
 Each part can be developed separately using each tool's live preview (`serve` command).
 
 Eventually we will switch to a single tool (Jekyll?) for all the parts.
+
+## Website
+
+Go to website folder and use the following commands
+
+Build assets and run server: bundle exec jekyll serve --watch 
+
+Watch and auto-build stylesheets: gulp watch
```

Comment:
You don't need to run two commands. You can run `npm run dev` which takes care of both.

Reply:
I get an error on that:
ZsoltKoszkysMBP:website kocsmy$ npm run dev
npm ERR! Darwin 17.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "dev"
npm ERR! node v6.13.0
npm ERR! npm  v3.10.10

npm ERR! missing script: dev
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/kocsmy/Hanno/redash-website/website/npm-debug.log